src/pages/en/guides/troubleshooting.mdx

```diff
@@ -137,6 +137,10 @@ nodeLinker: "node-modules"
 
 In Astro, using a `<head>` tag works like any other HTML tag: it does not get moved to the top of the page or merged with the existing `<head>`. Because of this, you usually only want to include one `<head>` tag throughout a page. We recommend writing that single `<head>` and its contents in a [layout component](/en/core-concepts/layouts/).
 
+### An unexpected `<script>` or `<style>` is included
+
+You may notice an imported component's `<script>` or `<style>` tags included in your HTML source even if that component doesn't appear in the final output. For example, this will occur with [conditionally rendered](https://docs.astro.build/en/core-concepts/astro-components/#dynamic-html) components that are not displayed. Astro's build process works on the module graph: once a component is included in the template, its `<script>` and `<style>` tags are processed, optimized, and bundled, whether it appears in the final output or not. This does not apply to scripts when the `is:inline` directive is applied.
```

Comment:
```suggestion
You may notice an imported component's `<script>` or `<style>` tags included in your HTML source even if that component doesn't appear in the final output. For example, this will occur with [conditionally rendered](https://docs.astro.build/en/core-concepts/astro-components/#dynamic-html) components that are not displayed. 

Astro's build process works on the module graph: once a component is included in the template, its `<script>` and `<style>` tags are processed, optimized, and bundled, whether it appears in the final output or not. This does not apply to scripts when the `is:inline` directive is applied.
```
Just added a paragraph break! LGTM!